2020/BillSplitter/BillSplitter/Data/BillsDbAccessor.cs

```diff
@@ -14,20 +15,30 @@ public BillsDbAccessor(BillContext context)
 
         public void AddBill(Bill bill)
         {
-            _context.Bill.Add(bill);
+            _context.Bills.Add(bill);
             _context.SaveChanges();
         }
 
+        public List<Bill> GetBillsByCustomerUserId(int userId)
```

Comment:
That makes accessors much bigger and violates SR principle. It is very specialized operation, maybe move it into another class.